### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Probhatmht**

### CODE REVIEW

File: cont.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+$conn = mysqli_connect("localhost","root","","php2");
+if(isset($_POST['sbtn'])){
+$email = $_POST['email'];
+$password = $_POST['password'];
```
Comment (Probhatmht) [inline cont.php:5]: need something ?

Comment (Jibanandamht) [reply cont.php:5]: you need space

**Review by Jibanandamht — CHANGES_REQUESTED** [review]
done?

File: update.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<?php
+include "cont.php";
+$id = $_GET['update_id'];
+$fetch_sql = "SELECT * FROM `users` WHERE id=$id";
+$fecth_query = mysqli_query($conn,$fetch_sql);
+$row = mysqli_fetch_assoc($fecth_query);
+$email = $row['email'];
+$password = $row['password'];
+
+$conn = mysqli_connect("localhost", "root", "", "php2");
```
Comment (Jibanandamht) [inline update.php:10]: here also .

**Review by Jibanandamht — CHANGES_REQUESTED** [review]

File: update.php (hunk 2 above)
Comment (Probhatmht) [reply update.php:10]: ok

**Review by Jibanandamht — COMMENTED** [review]
done bro

**Comment by Probhatmht** [comment]
ya

**Pull request merged**